pages/learn/deploy/deploy-with-balena-button.md

```diff
@@ -74,18 +74,18 @@ data:
 version: 3.5.2
 ```
 
-- `type` - Required field. In most cases this would be `sw.application`, unless you are implementing a block, in which case you need to use `sw.block`.
-- `name` - A user-friendly name of your application.
-- `description`: A description of what the application does. This is what is displayed if the application is published on [balenaHub][balenahub].
+- `type` - Required field. In most cases this would be `sw.fleet`, unless you are implementing a block, in which case you need to use `sw.block`.
```

Comment:
```suggestion
- `type` - Required field. In most cases this would be `sw.application`, unless you are implementing a block, in which case you need to use `sw.block`.
```